pipeline/errors/error_redirect.go

```diff
@@ -66,3 +68,19 @@ func (a *ErrorRedirect) Config(config json.RawMessage) (*ErrorRedirectConfig, er
 func (a *ErrorRedirect) GetID() string {
 	return "redirect"
 }
+
+func (a *ErrorRedirect) RedirectUrl(r *http.Request, c *ErrorRedirectConfig) string {
+	if c.UrlParam == "" {
+		return c.To
+	}
+
+	url, err := url.Parse(c.To)
+	if err == nil {
+		query := url.Query()
+		query.Set(c.UrlParam, r.URL.String())
+		url.RawQuery = query.Encode()
+		return url.String()
+	} else {
+		return c.To
+	}
```

Comment:
You usually want to first check the error and return immediately if it is not nil.
Therefore one also omits `else`.
Also consider shortening the name of `query` to `q` as it reflects the local usage.

```suggestion
	if err != nil {
		return c.To
	}
	q := u.Query()
	q.Set(c.UrlParam, r.URL.String())
	u.RawQuery = q.Encode()
	return u.String()
```